### ISSUE TITLE
check $PATH when installing with '-g'

### ISSUE THREAD

**Issue by mcollina** [npm/npm#6993]
When installing with '-g' NPM should check if the 'global' bin folder is in the $PATH, and propose a fix to the user. Given that NPM and node can be installed in so many different ways, it can definitely help newbies. See https://github.com/mcollina/mosca/issues/206 for more details.

**Comment by natpen** [comment]
@othiym23 : I wrote a little module to do just what's being described here! https://www.npmjs.com/package/is-path-member

**Comment by ariporad** [comment]
I'll implement this. (There is a high chance that in about 5 minutes, I'll post another comment being completely lost. You have been warned :smile:)

**Comment by ariporad** [comment]
Ok, I'm back:

I'm having trouble figuring out how to get the path that the module was installed to.

I'm pretty sure I want to do something like:
```javascript
if (npm.config.get('global')) {
    // Check for $PATH and warn if needed
}
```
[here](https://github.com/npm/npm/blob/master/lib/install.js#L291), and I think the path I'm looking for it `this.where`, but I'm not sure.

Thanks!

**Comment by ariporad** [comment]
Update: The path I'm looking for is *_not_* `this.where`, `this.where` is different from what I get when I run (for example) `which yo`.

**Comment by ariporad** [comment]
Never mind, I figured it out.

Fixed in #10190.

@othiym23: Could I convince you to look at/merge #10190? Thanks!

**Comment by othiym23** [comment]
#10190 still needs a test, but is otherwise ready to merge. If @ariporad doesn't have the time to come back and finish it, then I suppose one of the CLI team will. This is still a useful, simple thing to have in the CLI!
